Assets/Scripts/Utils/Utilities.cs

```diff
@@ -12,4 +13,17 @@ public static int countDigitsInt( int num )
 
         return ( numDigits );
     }
+
+    public static bool containsMeat( List<PizzaOrder.MeatItem> meats, Constants.Meats meatType )
+    {
+        foreach ( PizzaOrder.MeatItem meatItem in meats )
+        {
+            if ( meatItem.meatType == meatType )
+            {
+                return true;
+            }
+        }
+
+        return false;
+    }
```

Comment:
Can replace this whole function with just
```meats.Find(meatItem => meatItem.meatType == meatType)  != nullptr```
May want to do this rather than create a new function. See, C# has some cool features!!